src/pages/SubmissionConfig.js

```diff
@@ -125,19 +127,26 @@ export const SubmissionConfig = ({ methodId }) => {
         ])
       ]),
       div({ style: { marginTop: '2rem', height: '2rem', fontWeight: 'bold' } }, ['Select a data table']),
-      h(Select, {
-        isDisabled: false,
-        'aria-label': 'Select a data table',
-        isClearable: false,
-        value: selectedRecordType ? selectedRecordType : null,
-        onChange: ({ value }) => {
-          setSelectedRecordType(value)
-          loadRecordsData(value)
-        },
-        placeholder: 'None selected',
-        styles: { container: old => ({ ...old, display: 'inline-block', width: 200 }) },
-        options: _.map(t => t.name, recordTypes)
-      }),
+      div({}, [
+        h(Select, {
+          isDisabled: false,
+          'aria-label': 'Select a data table',
+          isClearable: false,
+          value: selectedRecordType ? selectedRecordType : null,
+          onChange: ({ value }) => {
+            setNoRecordTypeData(null)
+            setSelectedRecordType(value)
+            loadRecordsData(value)
+          },
+          placeholder: 'None selected',
+          styles: { container: old => ({ ...old, display: 'inline-block', width: 200 }), paddingRight: '2rem' },
+          options: _.map(t => t.name, recordTypes)
+        }),
+        noRecordTypeData && h(Fragment, [
+          a({ style: { marginLeft: '1rem', fontSize: 15, marginTop: '1rem', height: '2rem', fontWeight: 'bold' } }, [statusType.failed.icon(), ' ', noRecordTypeData])
```

Comment:
TOL: I am wondering if it would be better to display an orange `!` icon like below one instead of red triangle one? 🤔 
![Screen Shot 2023-01-24 at 6 06 15 PM](https://user-images.githubusercontent.com/16748522/214440929-5541593c-7de9-4a51-83e9-5e91d4fae989.png)

Reply:
I considered this. I chose the red warning since this appears due to an error in fetching from the WDS database. The orange `!` just seemed like a warning rather than an actual blocker. I don't mind changing it around however.

Reply:
I think it would be good to check with Sara or Lou.